src/frontend/src/views/Upload/components/Metadata/components/StaticTable/components/Row/style.ts

```diff
@@ -78,3 +83,31 @@ export const PrivateContent = styled.div`
 export const StyledLockIconWrapper = styled.div`
   ${rightMarginXxs}
 `;
+
+export const StyledAlertText = styled.p`
+  vertical-align: middle;
+  ${fontBodyXxxs}
+
+  ${(props: CommonThemeProps) => {
+    const colors = getColors(props);
+
+    return `
+      color: ${colors?.error[500]};
+      padding: 0 0 0 0;
```

Comment:
when using padding, giving a single integer applies that value to all 4 top bottom left right.
```suggestion
      padding: 0;
```